index.html

```diff
@@ -1238,6 +1238,369 @@ <h2>Evidence</h2>
+	"issuer": "https://dmv.example.gov/issuers/14",
+	"issued": "2010-01-01",
+	"delegatable": false,
+	"claim": {
+		"id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+		"ageOver": 21,
+	},
+	"proof": { ...
+	}
+}
+        </pre>
+
+       </section>
+       
+       <section>
+
+     <h3> Delegation through Recursion </h3>
+
+    <p>
+With recursion, the subject issues a credential to the delegated holder in which: 
+ the issuer is the subject, the claim contains 
+the id of the delegated holder, and the claim property is the credential that 
+was originally issued to the subject. An example is provided below.
+    </p>
+<p>
+Note: the exact format of this credential has not yet been agreed by the working group
+</p>
+ <pre class="example nohighlight" title="An example of a credential delegated 
+from a subject to a holder that is valid for 2 days">
+
+{
+	"id": "1234",
+	"type": ["Credential", "EmbeddedDelegatedCredential"],
+	"issuer": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+	"issued": "2010-01-06T00:00:00Z ",
+	"expires": "2010-01-08T00:00:00Z ",
+	"claim": {
+		"id": "did:example:ebfeb1f712ebc6f1c2123456789",
+		"credential": {
+			"id": "http://example.gov/credentials/3732",
+			"type": ["Credential", "Role"],
+			"issuer": "https://dmv.example.gov",
+			"issued": "2010-01-01",
+			"claim": {
+				"id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+				"role": "Project Manager"
+			},
+			"revocation": {
+				"id": "http://example.gov/revocations/738",
+				"type": "SimpleRevocationList2017"
+			},
+			"proof": {
+				"type": "LinkedDataSignature2015",
+				"created": "2016-06-18T21:19:10Z",
+				"creator": "https://example.com/jdoe/keys/1",
+				"domain": "json-ld.org",
+				"nonce": "598c63d6",
+				"signatureValue": "BavEll0...W3JT24 = "
+			}
+		}
+	},
+	"proof": { ...
+	}
+}
+
+        </pre>
+
+
+  </section>
+
+  <section>
+   <h3> Delegation through Verifiable Presentations </h3>
+
+   <p>
+
+Through the use of verifiable presentations, the subject delegates a credential to a
+holder, by issuing a new credential to the holder in which: the issuer is the subject, 
+the subject is the delegated holder, and the claim contains the property to be delegated. In addition, the delegated holder creates a verifiable presentation that contains these two credentials.
+</p>
+
+
+
+        <pre class="example nohighlight" title="An example of delegation using a 
+verifiable presentation">
+{
+	"id": "did:example:76e12ec21ebhyu1f712ebc6f1z2",
+	"type": ["VerifiablePresentation"],
+	"credential": [{
+			"id": "http://example.gov/credentials/3732",
+			"type": ["Credential", "Role"],
+			"issuer": "https://dmv.example.gov",
+			"issued": "2010-01-01",
+			"claim": {
+				"id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+				"role": "Project Manager"
+			},
+			"revocation": {
+				"id": "http://example.gov/revocations/738",
+				"type": "SimpleRevocationList2017"
+			},
+			"proof": {....}
+		},
+		{
+			"id": "https://some.dns.com/VC/123456789",
```

Comment:
W3C specs are required to use example.com and example.org in examples. dns.com is a real website.